cmd/podman-mac-helper/service.go

```diff
@@ -80,6 +83,17 @@ func service() int {
 		return 3
 	}
 
+	path := os.Getenv("PATH")
+	path = strings.Join([]string{path, "/usr/bin", "/usr/local/bin"}, ":")
+	os.Setenv("PATH", path)
+	cmd := exec.Command("docker", "context", "use", "default")
+
+	// Run the command
+	err = cmd.Run()
+	if err != nil && !errors.Is(err, exec.ErrNotFound) {
+		fmt.Print(fail)
```

Comment:
Question: I am largely unfamiliar with this helper but it seems strange to suppress the errors.

What happens if `errors.Is(err, exec.ErrNotFound)`?  Is it intentional to return success?

Reply:
I added this check because docker binary not existing in any of the paths might not be considered an error but on second thought, maybe it shouldn't be suppressed.

Reply:
I don't see how this can even work, this is a privileged helper AFAICT so you would execute docker as root here and not in the correct user context. If this is really need AFAICT it must happen in pkg/machine/qemu were it connects to the claim helper. cc @n1hility

Reply:
Right we should avoid running this as root. It needs to be done outside of the privileged helper, and ran on behalf of the user's account. It should live in the podman machine docker sock network setup logic